src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -16,8 +16,10 @@ import {
   TouchableOpacity,
   View
 } from 'react-native';
-import {useRoute} from '@react-navigation/core';
+import {useNavigation, useRoute} from '@react-navigation/core';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useNavigation` hook has been added. This is a good practice as it allows for navigation between different screens in the application.
```diff
+ import {useNavigation, useRoute} from '@react-navigation/core';
- import {useRoute} from '@react-navigation/core';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->